### DISCUSSION TITLE
https://angular.love/angular-zmienna-szablonu-let-hit-czy-kit

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#262] [General]
# https://angular.love/angular-zmienna-szablonu-let-hit-czy-kit

Angular.love - ciekawostki oraz rozwiązania dla średnio-zaawansowanych oraz zaawansowanych developerów Angulara.

https://angular.love/angular-zmienna-szablonu-let-hit-czy-kit

**Comment by DDonochVA** [comment]
Great article!
